### PR TITLE
Add change log for 2.2.1 release

### PR DESCRIPTION
**Pull request by Jiawei0227**
**What type of PR is this?**
/kind documentation

**What this PR does / why we need it**:
Add change log for 2.2.1 release

**Which issue(s) this PR fixes**:

Fixes #

**Special notes for your reviewer**:

**Does this PR introduce a user-facing change?**:

```release-note
NONE
```

/cc @msau42

### CODE REVIEW

File: CHANGELOG-2.2.md [hunk 1]
```diff
@@ -1,3 +1,26 @@
+# Release notes for v2.2.1
+# Changelog since v2.2.0
+
+## Changes by Kind
```
Comment (msau42) [inline CHANGELOG-2.2.md:4]: We're missing the note about the metrics?

Comment (Jiawei0227) [reply CHANGELOG-2.2.md:4]: The notes is included in the changed section where you can see the csi-lib-utils lib upgrade.
I am not sure if we want to separately add another note here.

Comment (jsafrane) [reply CHANGELOG-2.2.md:4]: From the changelog it's not really clear *why* we needed a new release. Is it just the metrics? Or release-tools update?

Comment (Jiawei0227) [reply CHANGELOG-2.2.md:4]: The intention is the metric fix. But it is already bound into the release-tools update.

**Pull request merged**